examples/file-transfer/package.json

```diff
@@ -62,5 +62,6 @@
     "example",
     "file",
     "upload"
-  ]
+  ],
+  "copyright.owner": "IBM Corp."
```

Comment:
Can we use somewhat consistent ordering of package.json fields across all packages please? I see that other packages have `copyright.owner` right after the `author` field, can we do the same everywhere you are adding `copyright.owner`?